crates/fiber-lib/src/cch/actor.rs

```diff
@@ -532,292 +438,89 @@ impl CchState {
             .as_ref(),
         )
         .into();
+
+        // Verify invoice UDT type script matches configured wrapped_btc_type_script
```

Comment:
if invoice.udt_type_script() is none,should return WrappedBTCTypescriptMismatch too

Reply:
Good catch